scripts/trigger-manual-deploy.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+echo "Starting Auto Manual Start workflows ..."
+
+github_token="$1"
+
+project="$2"
+
+project=$(echo "$project" | tr '[:lower:]' '[:upper:]')
+if [[ $project == "SS" ]]; then
+  project="SDS"
+fi
+
+# environment list: sbox, ptlsbox, ithc, ptl, stg, demo, test, dev
+environment="$3"
+
+# three parameters are required to trigger the workflow: github token, work area (sds, cft),
+# environment(sandbox, aat/staging, demo, ithc, ptl, etc). The cluster is set 00.
+
+# github token is $1 and work_area is $2 and environment is $3
+function trigger_workflow() {
+  echo "Workflow params: $1 $2 $3"
+
+  # Project: SDS or CFT or PANORAMA; SELECTED_ENV: sbox, test/perftest, ptlsbox, ithc, ptl, aat/staging, demo, test, preview/dev;
+  curl -L \
+         -X POST \
+         -H "Accept: application/vnd.github+json" \
+         -H "Authorization: Bearer $1" \
+         -H "X-GitHub-Api-Version: 2022-11-28" \
+         https://api.github.com/repos/hmcts/auto-shutdown/actions/workflows/manual-start.yaml/dispatches \
+         -d "{ \"ref\": \"master\",
+                \"inputs\": {
+                  \"PROJECT\": \"$2\",
+                  \"SELECTED_ENV\": \"$3\",
+                  \"AKS-INSTANCES\": \"00\"
+                }
+              }"
+}
+
+# github token is $1 and work_area is $2 and environment is $3
+function start_unhealthy_environments() {
+project_url="plum" && [[ "${project}" == "SDS" ]] && project_url="toffee"
+env="sandbox" && [[ "${environment}" != "sbox" ]] && env="ithc"
```

Comment:
This will lead to similar issues, can it be updated to only alter the env var if it's sbox -- if environment is perftest for instance, it would be set to ithc unless you came and changed this line too